This file would be ignored since GitCracken v8.2.0.
There is a new patcher with pattern match now.
See patchDirWithPro in src/patcher.ts for more detail.
--- src/main/static/startMainProcess.js
+++ src/main/static/startMainProcess.js
@@ -1,4 +1,19 @@
 /* eslint-disable no-console */
+
+function patchSnapshot() {
+  var RegistrationHelpers = snapshotResult.customRequire('./src/sharedModules/registration/shared/RegistrationHelpers.js');
+  var { decodeBody } = RegistrationHelpers;
+  RegistrationHelpers.decodeBody = function() {
+    var body = decodeBody(...arguments);
+    body = {
+      ...body,
+      licensedFeatures: ['pro'],
+      proAccessState: {}
+    }
+    return body;
+  }
+}
+
 (function() {
   if (process.env.GITKRAKEN_OPEN_MAIN_DEV_TOOLS) {
     require('./startDevTools');
@@ -32,6 +47,7 @@
   global.mode = buildMode;
 
   if (typeof snapshotResult !== 'undefined') {
+    patchSnapshot();
     const path = require('path');
     snapshotResult.setGlobals(global, process, global, {}, console, require, path.resolve());
     snapshotResult.customRequire('./src/main/main.js');
